core/src/main/kotlin/com/getstrm/pace/processing_platforms/bigquery/BigQueryTransformer.kt

```diff
@@ -38,6 +40,29 @@ object BigQueryTransformer : ProcessingPlatformTransformer(BigQueryRenderer) {
             )
         }
 
+    override fun hash(
+        field: DataPolicy.Field,
+        hash: DataPolicy.RuleSet.FieldTransform.Transform.Hash
+    ): Field<*> {
+        val dataType = field.toJooqField().dataType
```

Comment:
How well do the hash transforms handle the case of dbt where types are not explicitly part of the field definition?

Reply:
for now, we will require a dbt hash to have the data type included in the schema